msprime/trees.py

```diff
@@ -343,8 +349,12 @@ def draw(self):
                     dy=dy))
             if v != NULL_NODE:
                 y = self._x_coords[v], self._y_coords[v]
-                lines.add(dwg.line(x, (x[0], y[1])))
-                lines.add(dwg.line((x[0], y[1]), y))
+                if self._colour.get(v):
```

Comment:
OK - want to check for None too, so it would have to be `if v in self._colour and self._colour[v] is not None`